src/test/java/api/support/fixtures/ServicePointsFixture.java

```diff
@@ -86,4 +87,13 @@ public IndividualResource cd6()
 
     return servicePointRecordCreator.createIfAbsent(basedUponCircDesk6());
   }
+
+  public IndividualResource create(ServicePointBuilder builder)
```

Comment:
Can the other methods in this fixture can use this method instead of the record creator directly?

Reply:
Sure, I'll make that change.